examples/gno.land/r/sys/validators/validators.gno

```diff
@@ -1,60 +1,80 @@
-// Package validators is used to manage the validator set.
 package validators
 
 import (
 	"std"
 	"strconv"
 
-	"gno.land/p/gov/proposal"
+	"gno.land/p/sys/validators"
 )
 
-var unappliedChanges = []Change{}
-
-// Change represents a change in the validator set.
-type Change struct {
-	Address std.Address
-	Power   int
+// valset is the wrapper for the validator set protocol
+type valset struct {
+	p       validators.ValsetProtocol // p is the underlying validator set protocol
+	changes []validators.Validator    // changes are the set changes that happened between scrapes
 }
 
-// NewPropExecutor creates a new executor that wraps a changes closure
-// proposal. It emits a typed object (subscribed by tm2) only if it passes
-// through a complete p/gov/proposal process.
-func NewPropExecutor(changesFn func() []Change) proposal.Executor {
-	if changesFn == nil {
-		panic("changesFn should not be nil")
+// v holds the active on-chain validator set state
+var v *valset
```

Comment:
```suggestion
var (
	p       validators.ValsetProtocol // p is the underlying validator set protocol
	changes []validators.Validator    // changes are the set changes that happened between scrapes
)
```

Reply:
This is a personal preference nit, but I've changed it:

[fa341a6](https://github.com/gnolang/gno/pull/2130/commits/fa341a685730eb52fd7e3681d6f682afdf6c80f3)